baldrick/plugins/github_pull_requests.py

```diff
@@ -131,8 +141,61 @@ def process_pull_request(repository, number, installation, action,
             result = function(pr_handler, repo_handler)
             # Ignore skipped checks
             if result is not None:
+                # Map old plugin keys to new checks names.
+                # It's possible that the hook returns {}
+                for context, check in result.items():
+                    if check is not None:
+                        title = check.pop('description', None)
+                        title = check.pop('title', title)
+                        if title:
```

Comment:
I don't get this one. It will issue a warning even if `'description'` is `None` but `'title'` is given, no? Did I miss something?

I think the warning should be emitted if `'description'` is used regardless, like how we deal with deprecation in `astropy`.